packages/RPM/SPECS/fledge.spec

```diff
@@ -15,7 +15,11 @@ VCS:           __VCS__
 %define install_path	/usr/local
 
 Prefix:        /usr/local
+%if 0%{?centos} < 9 || 0%{?rhel} < 9
```

Comment:
python3-numpy ?

Log a Jira for CentOS stream 9 support, and test image datapoints etc.